maintainers/data/release-credits-handle-to-name.json

```
@@ -118,7 +118,7 @@
     "wh0": null,
     "mupdt": "Matej Urbas",
     "momeemt": "Mutsuha Asada",
-    "dwt": "\u202erekc\u00e4H nitraM\u202e",
+    "dwt": "Martin Häcker",
```

```suggestion
    "dwt": "Martin H\u00e4cker",
```
oh oops looking at the others it should be this